src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -1144,6 +1144,46 @@ class VogelIPRDescription(CommonIPR):
     # [[[end]]] (checksum: 9934660f6467a25e11bfbe4c95f8faa7)
 
 
+@attr.s(frozen=True, slots=True)
+class ForchheimerIPRDescription(CommonIPR):
+    """
+    .. include:: /alfacase_definitions/ForchheimerIPRDescription.txt
+
+    .. include:: /alfacase_definitions/list_of_unit_for_pressure.txt
+    .. include:: /alfacase_definitions/list_of_unit_for_forchheimer_linear_productivity_index.txt
+    .. include:: /alfacase_definitions/list_of_unit_for_forchheimer_quadratic_productivity_index.txt
+    .. include:: /alfacase_definitions/list_of_unit_for_dynamic_viscosity.txt
+    .. include:: /alfacase_definitions/list_of_unit_for_dimensionless.txt
+    .. include:: /alfacase_definitions/list_of_unit_for_length.txt
+    .. include:: /alfacase_definitions/list_of_unit_for_permeability_rock.txt
+    .. include:: /alfacase_definitions/list_of_unit_for_nonDarcy_flow_coefficient.txt
+    """
+
+    calculate_coeff_option = attrib_enum(
+        type_=constants.ForchheimerCoefficientsOption,
```

Comment:
Nice!

Reply:
Just
```
    calculate_coeff_option = attrib_enum(
        default=constants.ForchheimerCoefficientsOption.ReservoirParameters,
    )
```
will have the same effect, when givin `default` the `type_` is ignored. Maybe we should at least check if when using both the match.

Reply:
Thanks. Now I'm follow your suggestion